department-of-veterans-affairs/va.gov-cms
# VACMS-9511 update maintenance page styles

## Description

Relates to #9511. Paired PR with this devops PR: https://github.com/department-of-veterans-affairs/devops/pull/11537

This updates the maintenance page that is displayed during deployments to match the updated sitewide alert styles.

## Testing done
`cp docroot/sites/default/settings/settings.deploy.inactive.php docroot/sites/default/settings/settings.deploy.active.php`

This will ensure that the `'VA_GOV_IN_DEPLOY_MODE=TRUE'` in our local environment which is the flag checked to display this particular maintenance page.

## Screenshots
Before:
![Screen Shot 2022-07-08 at 8 02 56 AM](https://user-images.githubusercontent.com/11279744/178022398-1207d7b0-b526-4ffb-9620-c9ee07358b96.png)

After:
![Screen Shot 2022-07-08 at 8 19 59 AM](https://user-images.githubusercontent.com/11279744/178022419-56873af4-e712-470b-8a34-40f9c7aca50d.png)


## QA steps
I am not sure how to trigger this maintenance page for tugboat. You can try running the `cp` command above from testing steps in the tugboat terminal, or pull the branch locally and do the same thing. Or take my word for it 😅 

- set `VA_GOV_IN_DEPLOY_MODE=TRUE` in the environment to trigger this maintenance page.
- See that the styles have been updated to match the shield style alert from #9723. (minus the icon, it is from font awesome and this maintenance page is very barebones, none of that js is included, or needs to be, imo.)

### Definition of Done


### Select Team for PR review
